components/formats-gpl/test/matlab/TestBfopen.m

```diff
@@ -30,6 +31,13 @@
         filepath
         data
         nSeries = 1
+        flag = 1
+        width = 10;
+        height = 10;
+        x = 10;
+        y = 10;
+        DefaultSize = [512,512];
```

Comment:
This should already be given by `sizeX` / `sizeY` properties initialized in `ReaderTest`